scripts/darken_colors.lua

```diff
@@ -0,0 +1,164 @@
+-- Plugin lifepillar/vim-colortemplate must be available on &runtimepath
+--
+-- The current working directory must be set to the repo root
+
+-- IMPORTANT: the `cterm_color` key must always be below the `color` key of the
+--            same icon. Currently they are.
+
+-- This file should be run from the shell with `make colors`
+
+local fn = vim.fn
+
+if not jit then
+  error("Neovim must be LuaJIT-enabled to source this script")
+end
+
+if fn.filereadable("lua/nvim-web-devicons.lua") == 0 then
+  error("lua/nvim-web-devicons.lua not found")
+end
+
+if not vim.o.runtimepath:find("vim%-colortemplate") then
+  error("lifepillar/vim-colortemplate must be present in the runtimepath.")
+end
```

Comment:
Not happy when vim-colortemplate is at `~/.local/share/nvim/site/pack/packer/start/vim-colortemplate`.

How about:
```
local rc, err = pcall(vim.fn["colortemplate#colorspace#approx"], "#000000")
if not rc then
  error(err .. "\nPlease ensure lifepillar/vim-colortemplate present in the runtimepath.")
end
```